spot-ml/src/main/scala/org/apache/spot/lda/SpotLDAWrapper.scala

```diff
@@ -39,6 +39,21 @@ import scala.collection.immutable.Map
 
 object SpotLDAWrapper {
 
+  /**
+    * Runs Spark LDA and returns a new model.
+    *
+    * @param sparkSession       the SparkSession
+    * @param docWordCount       RDD with document list and the word count for each document (corpus)
+    * @param topicCount         number of topics to find
+    * @param logger             application logger
+    * @param ldaSeed            LDA seed
+    * @param ldaAlpha           LDA alpha parameter
+    * @param ldaBeta            LDA beta parameter
+    * @param ldaOptimizerOption LDA optimizer, em or online
+    * @param maxIterations      number of maximum iteration LDA will run
```

Comment:
Should this say, "maximum number of iterations for the optimizer" or something similar? The current line is a little unclear to me.